src/widgets/copilot/components/Messages/error-message.scss

```
@@ -0,0 +1,48 @@
+@import "src/css/colors.module.scss";
```

Stylelint (17.7.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)